packages/inventory/src/utils.ts

```diff
@@ -42,3 +44,29 @@ export const canDeleteStocktake = (row: StocktakeRowFragment): boolean =>
 
 export const isStocktakeDisabled = (row: StocktakeRowFragment): boolean =>
   row.status !== StocktakeNodeStatus.New || row.isLocked;
+
+export const stocktakesToCsv = (
```

Comment:
it is possible to simply throw the `data.nodes` into the `unparse` but this allows the download to match the list view more closely, update header values ( otherwise we have object prop names ) and exclude unwanted props ( __typename, and the pricing object.. for example)